mobify/bellows
# Upgrading to plugin 2.1.0

Bumping plugin version to latest

Status: **Ready to merge**

Reviewers: @tedtate @kpeatt 
## Changes
- bumped plugin version in bower.json
## How to test-drive this PR
- checkout this branch
- `bower install`
- `grunt test`
- `grunt serve` and make sure stuff isn't broken
